packages/core/src/node/backend-application-module.ts

```diff
@@ -85,4 +86,7 @@ export const backendApplicationModule = new ContainerModule(bind => {
 
     bind(WsRequestValidator).toSelf().inSingletonScope();
     bindContributionProvider(bind, WsRequestValidatorContribution);
+
+    bind(MessagingListener).toSelf().inSingletonScope();
+    bindContributionProvider(bind, MessagingListenerContribution);
```

Comment:
Lastly, I think we can move this binding to the messaging container: `packages\core\src\node\messaging\messaging-backend-module.ts`

Reply:
I’ll pick this up later! Thanks for your help. I didn’t run the linter before I committed, sorry.